src/main/java/org/sagebionetworks/web/client/widget/discussion/DiscussionThreadListItemWidget.java

```diff
@@ -39,12 +40,12 @@ public void configure(DiscussionThreadBundle bundle) {
 		this.bundle = bundle;
 		view.setTitle(bundle.getTitle());
 		view.setThreadUrl(TopicUtils.buildThreadLink(bundle.getProjectId(), bundle.getId()));
-		authorWidget.setTextHidden(true);
+		authorWidget.setBadgeType(BadgeType.SMALL_CARD);
 		authorWidget.configure(bundle.getCreatedBy());
 		view.clearActiveAuthors();
 		for (String userId : bundle.getActiveAuthors()) {
 			UserBadge user = ginInjector.getUserBadgeWidget();
-			user.setTextHidden(true);
+			user.setBadgeType(BadgeType.AVATAR);
```

Comment:
This looks kind of odd in production, here it is after changes: 

![image](https://user-images.githubusercontent.com/17580037/115630981-f90ad280-a2d2-11eb-833e-710e22b6a7b5.png)